Tilers/kd_tree.py

```diff
@@ -0,0 +1,31 @@
+from Tilers.object_to_tile import ObjectsToTile
+
+
+def kd_tree(objects, maxNumObjects, depth=0):
+    # objects can be of type objs or citym_cityobjects, herited from
```

Comment:
Instead of a comment, maybe place some test on the type of the
provided argument ? 
Additionally this type constraint placed on the arguments should
probably NOT depend on citym_cityobjects that is a specific implementation
(that kd_tree should not know about: otherwise you might have a circular
dependency or more simply you pollute the kd_tree general class with all
the specific implementations your might derive from ObjectToTile.

Actually, this might be the real constraint to check: `if not isinstance(objects,list(Obj)):....` ???

Reply:
I added a strong constraint by checking if the type is ObjectsToTile because the type returned by this methods should be a list<ObjectsToTiles>. 

Done in [#4e7ecd4](https://github.com/VCityTeam/py3dtiles/pull/5/commits/4e7ecd4bef4e9dfa054808e06e3bbc263cc69b3d)

Reply:
Nicely done. Maybe to improve readability a sequence of the form
```
if not is_instance(ObjectsToTile):
   return None
<...>
```
would be easier to read (avoiding a long indented block when the
else block does nothing).

Reply:
Done in [#86d44b4](https://github.com/VCityTeam/py3dtiles/pull/5/commits/86d44b4f519e9fb8fcee8538642e38483ce64a81)